profile/README.md

```diff
@@ -1,15 +1,41 @@
 # Think big
 
-![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)
+![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)  
+ 
+&nbsp;
+&nbsp;
+&nbsp;
+
```

Comment:
```suggestion
```

I suggest removing these. I understand what you are trying to do in terms of spacing it and such, but this is a markdown file and how it gets rendered will usually just depend on what interface it's displayed in. Might be cleaner to just leave this stuff out. There's a certain level of austerity and straight-forwardness to these kind of README file that devs expect.